### DISCUSSION TITLE
External file loading problem with pdpipe import

### DISCUSSION THREAD

**Discussion by fedorkobak** [pdpipe/pdpipe#97] [Q&A] [answered]
Will start with example. I create `test.py` file in some dir, it may looks like this:

```
from sys import path
import pdpipe

path.append("<path to external folder>/some_external_folder")
import temp
```
In ".../some_external_folder" i have added `temp.py` file, which contains `'hello world'` printing. When I try to run first file i have a error:
```
Traceback (most recent call last):
  File "<first file folder path>/test.py", line 6, in <module>
    import temp
ModuleNotFoundError: No module named 'temp'
```

But when i rem `import pdpipe` program starts. What can cause this behavior and how to correct it?

**Comment by fedorkobak** [comment]
After a while I noticed that the error disappears if I put `from sys import path` below `import pdpipe`. Looks like `pdpipe` doing something wrong with `sys`.

**Comment by shaypal5** [comment]


**Reply by fedorkobak** [reply]
Does the same code as I showed at the message below starts on your environment?

**Comment by shaypal5** [comment]
@Dranikf 
Sorry, right you are! I can get it to reproduce.

Let me investigate and I'll find the way I'm abusing `sys`. Sorry! :)

**Comment by shaypal5** [comment, marked as answer]
In the meanwhile, this works:

```python
from sys import path
path.append("/Users/user/temp_folder/")
import pdpipe
import temp
```

I suggest you move to work like this for now, as I'm using `sys` in a very specific way which I'm still not sure of its effects, and it is crucial for the packge's API.
